gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
Why is `-Xmx64m` required?

Reply:
I don't know how that even got added, I'll remove it.

Reply:
I just ran the `gradle wrapper` task and that's what it generated automatically. /shrug